src/tests/utils.test.ts

```diff
@@ -1,4 +1,7 @@
+import { NonDeletedExcalidrawElement } from "../element/types";
+import * as exportUtils from "../scene/export";
```

Comment:
lets rename this file to `export.test.ts` instead to match the original file name.

Reply:
Alright! 👍